admin_guide/node-allocatable-resources.adoc

```diff
@@ -0,0 +1,58 @@
+= Node Allocatable resources
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+== Overview
+To provide more reliable scheduling and minimize node resource overcommitment,
+each node has a chance to specify how much resources get consumed
+by kubernetes related components (e.g. kubelet, kube-proxy, docker) running on a node
+and by remaining non-kubernetes components (e.g. system daemons, databases).
```

Comment:
```
and by remaining components (e.g. sshd, NetworkManager).
```
That is a question. My interpretation of ``--kube-reserved`` was covering docker as well [1]. I interpret ``--system-reserved`` as resource reserved for everything that is not kubernetes/openshift related. NetworkManager could be misinterpreted incorrectly this way but as it does not have to be used with k8s/openshift, it falls out of ``--kube-reserved``.

[1] https://github.com/timstclair/kubernetes/blob/54a6da979c6d85dc81fae364124d5c7dd465bc6c/docs/proposals/node-allocatable.png